tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -40,6 +40,11 @@ def stop_pfcwd(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     logger.info("--- Stop Pfcwd --")
     duthost.command("pfcwd stop")
 
+    yield
+
+    logger.info("--- Start Pfcwd --")
```

Comment:
Same as above, I think we can combine into one file. Also wondering why we have two different scopes